gui/templates/menubar.html

```diff
@@ -69,11 +69,30 @@
     <span id="menuBar_AlertText">{% trans "Alert" %}</span>
   </div>
 
-  <div id="menuBar_Documentation" data-dojo-type="dijit.MenuBarItem" onClick="Menu.openDocumentation();" class="menu_icon right">
-    <img src="{{ STATIC_URL }}images/ui/menu/{{ sw_name|lower }}guide.png" />
-    <br />
-    <span>{% trans "Guide" %}</span>
-  </div>
+  {% if sw_name|lower != "freenas" %}
+    <div id="menuBar_Documentation" data-dojo-type="dijit.PopupMenuBarItem" class="menu_icon right">
+      <span>
+        <img src="{{ STATIC_URL }}images/ui/menu/{{ sw_name|lower }}guide.png" />
+        <br />
+        Guide
+      </span>
+      <div data-dojo-type="dijit.DropDownMenu">
+          <div data-dojo-type="dijit.MenuItem" onClick="Menu.openDocumentation();">HTML Documentation</div>
```

Comment:
This should also say User Guide, because it's the same thing, just a different format:

User Guide (HTML)
User Guide (PDF)

Reply:
I don't know details about how translations are handled here, but ultimately we should try to show the translation for the locale set in the web GUI, falling back to the English version if a translated version is not available.

Reply:
@wblock I can fix the naming convention.
When it comes to the translation the FreeNAS code is going to be translated normally; the TrueNAS part is left without translation(IIRC we don't translate the TrueNAS part of the codebase anyway.)